robot_motion_planning_summary.tex

```diff
@@ -31,147 +32,457 @@ \section{Introduction}
 	\item robustness to disturbances
 	\item probability of success
 \end{itemize}
+\vspace{2.5mm}
+To avoid long calculations in online processes, often an offline
+pre-processing is done and a road-map is calculated. When
+the manipulator is high-complex it is not possible to calculate
+the road-map beforehand and thus probabilistic approaches are
+used. Those perform sampling and create the road-maps with
+uncertain data.
 
 \section{Configuration Space}
+\subsection{Basics}
+\textbf{Workspace:} Reachable space of the end-effector. \\
+\textbf{Task space:} Corresponds to the space where the robot's task can be naturally expressed. \\
+\textbf{Configuration:} The smallest number of real-valued coordinates to fully describe configurations of a robot. For example, a mobile robot moves in directions $x,y$ and can turn its yaw heading $\theta$ - a configuration of this robot could be $\mathbold{c} = (1,3,30^\circ)^\mathrm{T}$. \\
+\textbf{Configuration space (\textit{C-space}):} The space of all possible configurations, that a robot can have in a certain workspace. \\
+\textbf{Dimension of the C-space/DoF:} Minimum number of parameters needed to specify the configuration. Because there can also be angles in the topology of the C-space, it is usually not a Cartesian space (e.g. $2\pi = 4\pi$, cyclic repetition along angle axis). \\
+\textbf{Common robot models:} Single points, single rigid bodies and multiple rigid bodies with joints. \\
+\textbf{Non-holonomic robot:} Robot, that cannot move freely on the
+configuration space manifold because it is constrained in some way. For example, a car cannot drive sideways. \\
+\textbf{Path:} Continuous curve in configuration space $C$ connecting two configurations $q$ and $q'$:
+$\tau: s \in [0,1] \rightarrow \tau(s) \in C$ s.t. $\tau(0) = q$ and $\tau(1) = q'$\\
+\textbf{Trajectory:} A trajectory is a path parametrized by time:
+$\tau: t \in [0, T] \rightarrow \tau(t) \in C$.
+Possible constraints: Smoothness, minimum length, minimum time, ...
```

Comment:
No capitalization after the colon here. Please also check the other places in the text.
https://www.grammarbook.com/blog/commas/capitalization-with-colons/

Reply:
Changed it everywhere in the text where I could find it.